Abyss.Web/ClientApp/src/app/servermanager/servermanager.component.html

```diff
@@ -1,13 +1,16 @@
 <h1>Server Manager</h1>
+<h1 *ngIf="!loading && !hubReady">Lost connection to server manager hub, reconnecting...</h1>
```

Comment:
Same, does the user really care? If you are just going to auto-reconnect 🤷

Reply:
I've changed this since to pop a message in the log instead, it's there incase they are trying to start/stop a server and lose connection and it will get stuck so at least they know